migrations/mystery_items.js

```diff
@@ -27,6 +27,8 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
+  db.users.find({'purchased.plan.dateTerminated':{$exists:true}}).forEach(function(user) {
```

Comment:
@crookedneighbor I did not see that at first (my google-fu must be weak today!), thanks! Upon further investigation, though, it seems as if findAndModify will only modify a single document, regardless of how many the find operation returns. Considering the use of the multi option in the old update statement, would it be safe to guess that we need the ability to modify multiple docs at once?